app/utils/ui-kit/concat.js

```diff
@@ -0,0 +1,15 @@
+import { computed } from '@ember/object';
+
+export default function concat(...props) {
+  return computed(...props, function () {
+    const results = props.reduce((collection, prop) => {
+      const val = this.get(prop);
+      if (val !== undefined && val !== null) {
+        collection.push(val);
+      }
+      return collection;
+    }, []);
+
+    return results.join(' ');
```

Comment:
Just a little simplification suggestion, completely optional:
```js
export default function concat(...props) {
  return computed(...props, function () {
    return props.reduce((collection, prop) => [...collection, this.get(prop)], []).compact().join(' ');
  });
}
```

Reply:
I like it :+1: Change made